rcl/include/rcl/time.h

```diff
@@ -118,29 +119,38 @@ typedef struct rcl_jump_threshold_t
 /// Struct to describe an added callback.
 typedef struct rcl_jump_callback_info_t
 {
+  /// Callback to fucntion.
   rcl_jump_callback_t callback;
+  /// Threshold to decide when to call the callback.
   rcl_jump_threshold_t threshold;
+  /// Pointer passed to the callback.
   void * user_data;
 } rcl_jump_callback_info_t;
 
 /// Encapsulation of a time source.
 typedef struct rcl_clock_t
 {
+  /// Clock type
   enum rcl_clock_type_t type;
   /// An array of added jump callbacks.
   rcl_jump_callback_info_t * jump_callbacks;
   /// Number of callbacks in jump_callbacks.
   size_t num_jump_callbacks;
+  /// Pointer to get_now function
   rcl_ret_t (* get_now)(void * data, rcl_time_point_value_t * now);
   // void (*set_now) (rcl_time_point_value_t);
```

Comment:
Is this dead code? Feel free to remove it
```suggestion
  // void (*set_now) (rcl_time_point_value_t);
```